### PR TITLE
file watching using browser sync

### PR DESCRIPTION
**Pull request by esprush**
@manojVivek @jjavierdguezas hot reloading for local files using browserSync. added support for js,html and css.

### CODE REVIEW

File: desktop-app/app/utils/browserSync.js [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
 export async function initBrowserSync() {
-  if (!browserSync.active) {
+  if (!browserSyncEmbed.active) {
     await initInstance();
   }
 }
 
+export function watchFiles(fileDir) {
+  filesWatcher = browserSyncEmbed
+    .watch([`${fileDir}/*.html`, `${fileDir}/**/**.js`])
```
Comment (jjavierdguezas) [inline desktop-app/app/utils/browserSync.js:15]: @esprush 
If we are watching all `*.js` and all `*.css` in all sub folders, why not to do the same with `*.html`?

File: desktop-app/app/main.dev.js [hunk 2]
```diff
@@ -305,30 +307,19 @@ const createWindow = async () => {
     onResize();
   });
 
-  const watcher = new chokidar.FSWatcher();
```
Comment (jjavierdguezas) [inline desktop-app/app/main.dev.js:308]: should we uninstall `chokidar` package right?

File: desktop-app/app/utils/browserSync.js (hunk 1 above)
Comment (manojVivek) [reply desktop-app/app/utils/browserSync.js:15]: I think we should only watch the currently loaded `.html` file, as the other HTML files are not related to the current page.

Comment (manojVivek) [reply desktop-app/app/utils/browserSync.js]: @esprush  @jjavierdguezas Merging this now. Let's address any concerns in a new PR.

**Pull request merged**